README.pt-br.md

```diff
@@ -602,11 +620,207 @@ ObxValue(
 ),
 ```
 
-## Explicação em vídeo sobre Outras Features do GetX
-
+### Explicação em vídeo sobre Outras Features do GetX
 
 Amateur Coder fez um vídeo incrível sobre utils, storage, bindings e outras features! Link: [GetX Other Features](https://youtu.be/ttQtlX_Q0eU)
 
+
+## Dicas Úteis
+
+`.obs`ervables (também conhecidos como _Rx_ Types) possuem uma grande variedade de métodos e operadores internos.
+
+> É muito comum acreditar que uma propriedade com `.obs` **É** o valor real... mas não se engane!
+> Evitamos a declaração de tipo da variável, porque o compilador do Dart é inteligente o suficiente e o código
+> parece mais limpo, mas:
+
+```dart
+var message = 'Hello world'.obs;
+print( 'Message "$message" é do tipo ${message.runtimeType}');
+```
+
+Mesmo que `message` _imprima_ o valor da string, seu tipo é **RxString**!
+
+Então, você não pode fazer `message.substring( 0, 4 )`.
+Você tem que acessar o `valor` real dentro do _observable_:
+A "maneira" mais usada é utilizando `.value`, mas, você sabia que também pode usar:
+
+```dart
+final name = 'GetX'.obs;
+// apenas "atualiza" o stream, se o valor for diferente do atual.
+name.value = 'Hey';
+
+// Todas as propriedades Rx são "chamáveis" e retorna o novo valor.
+// mas esta abordagem não aceita `null`, a UI não será reconstruída
+name('Hello');
+
+// é como um getter, imprime 'Hello'
+name() ;
+
+/// números:
+
+final count = 0.obs;
+
+// Você pode usar todas as operações não mutáveis ​​das primitivas de num! 
+count + 1;
+
+// Cuidado! isso só é válido se `count` não for final, mas var
+count += 1;
+
+// Você também pode comparar com os valores:
+count > 2;
+
+/// booleans:
+
+final flag = false.obs;
+
+// mude o valor entre true/false
+flag.toggle();
+
+
+/// todos os tipos:
+
+// Defina `value` como null.
+flag.nil();
+
+// Todas as operações toString() e toJson() são passada para `value`
+print( count ); // chama `toString()` de RxInt
+
+final abc = [0,1,2].obs;
+// Converte o valor em um Array json, imprime RxList
+// Json é suportado por todos os Rx types!
+print('json: ${jsonEncode(abc)}, type: ${abc.runtimeType}');
+
+// RxMap, RxList e RxSet são Rx types especiais, que estendem seus tipos nativos.
+// mas você pode trabalhar com uma lista como uma lista normal, embora seja reativa!
+abc.add(12); // Coloca 12 na lista, e ATUALIZA o stream.
+abc[3]; // como uma lista lê o índice 3.
+
+// a igualdade funciona com o Rx e o valor, mas hashCode é sempre obtido do valor 
+final number = 12.obs;
+print( number == 12 ); // prints > true
+
+/// Rx Models personalizados:
+
+// toJson(), toString() são transferidos para o filho, para que você possa implementar 
+// override neles e imprimir o observável diretamente.
+
+class User {
+    String name, last;
+    int age;
+    User({this.name, this.last, this.age});
+
+    @override
+    String toString() => '$name $last, $age years old';
+}
```

Comment:
não entendi. 
quer dizer que eu posso fazer algo tipo assim?
```dart
var user = User().obs;
print(user.toString());
```

se for isso mesmo, poderia adicionar isso na documentação? O que mais falta no nosso doc são exemplos, então mesmo que seja uma linha a mais mostrando como usar, é bom

Reply:
Seu pensamento está certo, inclusive no comentário da linha 705 que está acima, indica que podemos imprimir o observável diretamente. Neste trecho são explicados este conceito e também o conceito de atualização de uma propriedade da classe e a notificação necessária. Ao final do código, na linha 713 já existe um print( user ), então poderíamos exemplificar o result dele assim:
`print( user ); // Resultado (toString): Roi Doe, 33 years old`
Fico um pouco preocupado de fazer uma mudança acentuada neste readme em português e ele não refletir o readme em inglês. O que acha?

Reply:
eu esqueci que dar print numa classe evoca o toString automaticamente,
uma linha só a mais do que o o inglês não faz mal, o problema é quando falta seções inteiras ou coisas assim

> Ao final do código, na linha 713 já existe um print( user ), então poderíamos exemplificar o result dele assim:
print( user ); // Resultado (toString): Roi Doe, 33 years old

boa ideia, se puder colocar isso por favor ficaria excelente